HW04.Rmd

```diff
@@ -29,33 +18,70 @@ Knitr is part of the knitr package. Allows you to change column names and add ca
 
 
 ```{r month, warning=FALSE}
+gun_deaths_per_month <- thedata %>%
+    select(month) %>%
+    group_by(month) %>%
+    count(month) %>%
+    rename(deaths_per_month = n)
 
+kable(gun_deaths_per_month, align = 'c', col.names = c("Month", "Deaths/Month"))
 ```
 
 
 ### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled "Jan", "Feb", "Mar" (full or abbreviated month names are fine), not `1`, `2`, `3`.
 
 ```{r month_chart}
+thedata$month <- factor(thedata$month, labels = c("January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"))
 
+ggplot(thedata, aes(x = month, fill = month)) +
+    geom_bar() + 
+    labs(x = "Month", y = "NUmber of Deaths") 
 ```
+## Is there a way to make my axis text smaller so they're not overlapping like this? Also, I'm going to try and make the graph slightly less colorful. As it stands I think it's a bit too flamboyant. 

```

Comment:
I just used the abbreviation for each month and that seemed to fit fine but I believe you should also be able to use element_text(angle =___) inside your theme call to change the angle for each label, for example: 

theme(axis.text.x = element_text(angle = 90)

to make the graph less colorful you could either set a specific fill color for all the bars or just remove the "fill = month" that is where the bars are changing color based on each month.  I have been refering to this PDF file for a color palette in R (http://www.stat.columbia.edu/~tzheng/files/Rcolor.pdf).